apps/extension/src/message/popup.ts

```diff
@@ -1,12 +1,12 @@
 import type { TransactionView } from '@buf/penumbra-zone_penumbra.bufbuild_es/penumbra/core/transaction/v1/transaction_pb';
 import type { AuthorizeRequest } from '@buf/penumbra-zone_penumbra.bufbuild_es/penumbra/custody/v1/custody_pb';
-import type {
+import {
   InternalMessage,
   InternalRequest,
   InternalResponse,
 } from '@penumbra-zone/types/src/internal-msg/shared';
-import type { Jsonified } from '@penumbra-zone/types/src/jsonified';
 import { UserChoice } from '@penumbra-zone/types/src/user-choice';
+import { Jsonified } from '@penumbra-zone/types/src/jsonified';
```

Comment:
i often used type imports to ensure they generated no actual code effect, we shouldn't convert type imports to normal imports